src/setup/rgw_start.py

```diff
@@ -27,8 +27,13 @@ class RgwStart:
     def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
-        except OSError as e:
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
+            for _ in range(SERVICE_RETRIES):
+                _, stderr, rc = SimpleProcess(cmd).run()
+                if rc != 0:
+                    Log.error(stderr)
+            else:
+                raise Exception
+        except Exception as e:
             Log.error(f"Failed to start radosgw service:{e}")
             raise SetupError(e.errno, "Failed to start radosgw service. %s", e)
```

Comment:
OS have e.errno. Since we have moved to SimpleProcess you need to update this with rc